modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -44,31 +44,24 @@ This section contains subsections for each data source.
 
 === Add local data file
 
-After clicking btn:[Local File], you will be prompted to choose a local file to upload through the GUI.
+After clicking btn:[Local File], you will be prompted to choose one or more local files to upload through the GUI.
 
 image::choose-a-local-file.png[]
 
-Click the Upload File button image:upload_file_btn.png[] and choose the data files you want to upload to the server data folder, then click btn:[ADD].
-
-image::upload-files.png[]
-
-If all the files are uploaded successfully, the progress window will close automatically, otherwise the window will stay open to notify users of any errors.
+If all the files are uploaded successfully, a message of success will appear at the bottom of the screen.
+After you see that message, you may click btn:[Next].
 
 Once the files are uploaded to the server, they will appear in the "Files on server" list on the left side of the Add Data Files window.
```

Comment:
The "Files on server" will only display the data format chosen by user in the previous step. If you upload a csv but choose json type, you won't see your csv file in the "Files on server". We should mention that to avoid confusion.